apps/pass-image-api/src/tiles.rs

```diff
@@ -82,6 +84,17 @@ async fn fetch_tile_box(
     top_left: &TileCoordinate,
     bottom_right: &TileCoordinate,
 ) -> Result<HashMap<(u32, u32, u32), Bytes>, Error> {
+    // Create a manual span for this function
+    // This span will be the parent of all outgoing calls
+    let tracer = global::tracer("fetch_image_tracer");
+    let span = tracer
+        .span_builder("fetch_image")
+        .with_kind(SpanKind::Internal)
+        .start(&tracer);
+
+    let cx = Context::current_with_span(span);
```

Comment:
There's lots of ceremony around this, but I can follow it and in terms of the borrowing it makes sense. And then of course you have to clone further on because  of the `async move`.